src/routes/user/index.js

```diff
@@ -28,8 +28,12 @@ export async function userRoutes(server) {
       const response = createSuccessfulResponse("post", req.body, token);
       res.code(HTTP_CREATED).send(response);
     } catch (err) {
+      const message = "Invalid Email or Password";
```

Comment:
This isn't a correct message. I mean usually if it's an invalid password or email we'd have to do some sort of validation to determine whether or not it is a password or not. Let's say a valid password is atleast 5 characters and it has to be alphanumeric. For an email you can use the regex here and validate an email against this: https://emailregex.com/ in javascript.